tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks/UsageReport/WriteUsageBurndownData.cs

```diff
@@ -0,0 +1,124 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Build.Framework;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using NuGet.Packaging.Core;
+using NuGet.Versioning;
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using System.Xml.Linq;
+using Task = Microsoft.Build.Utilities.Task;
+
+namespace Microsoft.DotNet.SourceBuild.Tasks.UsageReport
+{
+    public class WriteUsageBurndownData : Task
+    {
+        /// <summary>
+        /// Specifies the root directory for git.
+        /// </summary>
+        [Required]
+        public string RootDirectory { get; set; }
+
+        /// <summary>
+        /// Specifies the path to the prebuilt baseline file
+        /// to be used to generate the burndown.
+        /// </summary>
+        [Required]
+        public string PrebuiltBaselineFile { get; set; }
+
+        /// <summary>
+        /// Output data CSV file. 
+        /// </summary>
+        [Required]
+        public string OutputFilePath { get; set; }
+
+        class commit
+        {
+            public string sha { get; set; }
+            public string title { get; set; }
+            public DateTime commitDate { get; set; }
+            public int packageVersionCount { get; set; }
+            public int packageCount { get; set; }
+
+            public override string ToString()
+            {
+                return $"{sha}, {title}, {commitDate}, {packageVersionCount}, {packageCount}";
+            }
+        }
+
+        public override bool Execute()
+        {
+            string baselineRelativeFileName = PrebuiltBaselineFile.Replace(RootDirectory, "");
```

Comment:
It's normal for MSBuild, but I think needing the trailing `/` (right?) should be in the property doc.

Reply:
Added a comment.